tests/e2e/steps_start_chains.go

```diff
@@ -159,12 +159,6 @@ func stepsStartConsumerChain(consumerName string, proposalIndex, chainIndex uint
 					{Id: ValidatorID("alice"), Stake: 500000000, Allocation: 10000000000},
 					{Id: ValidatorID("carol"), Stake: 500000000, Allocation: 10000000000},
 				},
-				// For consumers that're launching with the provider being on an earlier version
```

Comment:
thanks for figuring out how to remove this.

Much appreciated!

Reply:
Thanks for pointing it out 😃